frontend/src/views/WheresMyGene/components/HeatMap/utils.ts

```diff
@@ -72,8 +73,10 @@ export function createChartOptions({
     },
     grid: {
       bottom: Y_AXIS_BOTTOM_PADDING,
-      left: Y_AXIS_CHART_WIDTH_PX + "px",
-      top: X_AXIS_CHART_HEIGHT_PX + "px",
+      top: "0px",
+      left: "0px",
+      width: `${heatmapWidth}px`,
+      height: `${heatmapHeight}px`,
```

Comment:
can we add a comment here to highlight that this is the key change to align y axis labels please? Thank you!